### PR TITLE
fix, ubr backup descriptor uses glob syntax

### PR DESCRIPTION
**Pull request by lsh-0**
yeah, yeah, I know

fyi @giorgiosironi

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Uh, cyclical dependency where the new certificate group customization from https://github.com/elifesciences/builder-base-formula/pull/143 needs on the Vault group. I'll add a `group.present` there.

**Comment by lsh-0** [comment]
sorry I didn't investigate further, I've neglected the non-article metrics this week and wanted to make a push on that

**Comment by giorgiosironi** [comment]
Now base still fails on:
```
[34.224.166.48] out:           ID: vault-user
[34.224.166.48] out:     Function: user.present
[34.224.166.48] out:         Name: vault
[34.224.166.48] out:       Result: False
[34.224.166.48] out:      Comment: Failed to create new user vault
[34.224.166.48] out:      Started: 13:43:34.573774
[34.224.166.48] out:     Duration: 33.547 ms
[34.224.166.48] out:      Changes:  
```

**Comment by giorgiosironi** [comment]
```
[34.224.166.48] out: [ERROR   ] stderr: useradd: group vault exists - if you want to add this user to that group, use -g.
```

**Comment by giorgiosironi** [comment]
`base` is broken, but `fresh` now fixes it.

File: salt/master-server/vault.sls [hunk 1]
```diff
@@ -83,7 +85,7 @@ vault-cli-client-environment-configuration:
 # can check on the first highstate is that a daemon is listening
 vault-smoke-test:
     cmd.run:
-        - name: nc -q0 -w1 -z localhost 8200
```
Comment (lsh-0) [inline salt/master-server/vault.sls]: kinda liked seeing the old school netcat make an appearance

**Pull request merged**